src/components/EventCard/EventCard.js

```diff
@@ -55,7 +55,9 @@ const EventCard = ({ event, className }) => {
           </Link>
         </div>
 
-        <RsvpBadge totalAccepted={totalAccepted} center />
+        { totalAccepted > 0 &&
```

Comment:
Can you put totalAccepted on the next line?

Reply:
Done!